parser/parser.y

```diff
@@ -989,6 +1010,27 @@ import (
 	ResumeLoadDataStmt         "RESUME LOAD DATA JOB statement"
 	CancelLoadDataStmt         "CANCEL LOAD DATA JOB statement"
 	DropLoadDataStmt           "DROP LOAD DATA JOB statement"
+	ProcedureUnlabeledBlock    "Procedure unlabel block info"
+	ProcedureBlockContent      "Procedure block info"
+	SimpleWhenCase             "Procedure when then"
+	ProcedureStatementIf       "Procedure if block info"
+	procedurceElseIfs          "Procedure else block info"
+	ProcedureIf                "Procedure if data info"
+	ProcedureStmtUnlabeled     "Procedure stmt unlabeled"
+	ProcedureUnlabeledControl  "Procedure while unlabeled"
+	ProcedureCaseStmt          "Procedure case test"
+	ProcedureSimpleCase        "Procedure case when"
+	ProcedureSearchedCase      "Procedure search when then"
+	ProcedureSelectStmt        "Can use cursor select"
+	ProcedureOpenCur           "Open procedurce cursor"
+	ProcedureCloseCur          "Close procedurce cursor"
+	ProcedureFetchInto         "Procedure fetch into cursor"
+	ProcedureHcond             "Procedure handle error condition"
+	ProcedurceCond             "Procedure handle error code"
```

Comment:
1. what's the difference of `handle error condition` and `handle error code`?
2. `ProcedurceCond` seems unreasonable.

Reply:
1. 
<img width="692" alt="image" src="https://user-images.githubusercontent.com/97376271/230709798-1a6cf609-bedb-4e8a-8e73-46d5651fc5bb.png">
error code is mysql_error_code && sqlstate_value
error condition is condition_value